src/api/spec/features/webui/users/users_notifications_settings_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "browser_helper"
+
+RSpec.feature "User's notifications settings", type: :feature, js: true do
+  let!(:user_with_groups) { create(:user_with_groups, login: 'moi_wg') }
```

Comment:
Now make sense... before was used in other specs inside this file (the were dropped)